packages/components/src/components/video/video.stories.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, args, parameters } = storybookDefaults('sd-video');
+const { generateTemplate } = storybookTemplate('sd-video');
+
+/**
+ * The `sd-video` component can be used to wrap external video elements in order to provide basic styling for Union Investment.
+ * ## Implementation
+ * Use in combination with a viewer component (e. g. from Moving Image) or a bare `<video>`-Tag in addition to `sd-media`.
+ * `<sd-media>
+ *  <sd-video><video></video></sd-video>
+ *  <p slot="copyright">This is mine!!!</p>
+ * </sd-media>`
+ */
+
+export default {
+  title: 'Components/sd-video',
+  component: 'sd-video',
+  args,
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+/**
+ * `sd-video` does not have any content by default because it is a wrapper component.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
```

Comment:
As mentioned here https://github.com/solid-design-system/solid/pull/451#issuecomment-1757026290 we should show at least an image in the main slot.